d2bs/kolbot/libs/bots/MFHelper.js

```diff
@@ -136,6 +136,10 @@ MainLoop:
 				delay(100 + me.ping);
 			}
 
+			if (me.inTown && (Config.HealHP > 0 && me.hp < Math.floor(me.hpmax * Config.HealHP / 100))) {
```

Comment:
Why not just if me.inTown and then call Town.heal()? also would put this before the while statement.